.github/workflows/ci-it.yaml

```diff
@@ -45,6 +45,8 @@ jobs:
           bash <(curl -s https://codecov.io/bash)
       - name: 'Check Dependencies Licenses'
         run: tools/dependencies/check-LICENSE.sh
+      - name: 'Check Javaagent Plugin List'
+        run: tools/plugin/check-javaagent-plugin-list.sh
```

Comment:
I suggest to move this to the beginning of the the steps (line 42) because this script is more lightweight that can fail quickly, and thus prevents unnecessary `Install & Test`

Reply:
Make sense, it could fail fast.